images/image.go

```diff
@@ -268,6 +268,9 @@ func Platforms(ctx context.Context, provider content.Provider, image ocispec.Des
 	var platformSpecs []ocispec.Platform
 	return platformSpecs, Walk(ctx, Handlers(HandlerFunc(func(ctx context.Context, desc ocispec.Descriptor) ([]ocispec.Descriptor, error) {
 		if desc.Platform != nil {
+			if desc.Platform.OS == "unknown" || desc.Platform.Architecture == "unknown" {
```

Comment:
Wondering if we should also update the function description to mention that such platforms are omitted; I just noticed that it's not called out, so a caller may expect the list to include any platform (including the `unknown/unknown` ones);

```go
// Platforms returns one or more platforms supported by the image.
func Platforms(ctx context.Context, provider content.Provider, image ocispec.Descriptor) ([]ocispec.Platform, error) {
```

Reply:
Since `unknown/unknown` is not really a platform supported by image, I think the current comment should be good.

I can create a separate PR to give a more detailed description in the comment, as this is opened from the cherry-pick bot.